### PR TITLE
Updated "pull from existing repo

### PR DESCRIPTION
**Pull request by dgallardox**

### CODE REVIEW

File: atlas/getting-started/deploy-from-existing-repo.mdx [hunk 1]
```diff
@@ -12,11 +12,9 @@ Atlas supports Node.js applications using popular frameworks like [Faust.js](htt
 
 ### 1. Prepare your app for deployment
 
-First make sure that you've pushed your application's code to your GitHub repository. If you're not familiar with pushing code to a GitHub repository, see this guide: [Adding an existing project to GitHub using the command line](https://docs.github.com/en/get-started/importing-your-projects-to-github/importing-source-code-to-github/adding-an-existing-project-to-github-using-the-command-line).
+When building the front-end app, WPEngine will use ```npm run build``` by default. If you want to specify a build command, you can do so in one of two ways:
```
Comment (kellenmace) [inline atlas/getting-started/deploy-from-existing-repo.mdx:15]: @dgallardox "WP Engine" should be two words-- need to add a space between them.
Also, \```npm run build\``` can be simplified to \`npm run build\`. Only one set of backticks is necessary for inline code.

File: atlas/getting-started/deploy-from-existing-repo.mdx [hunk 2]
```diff
@@ -12,11 +12,9 @@ Atlas supports Node.js applications using popular frameworks like [Faust.js](htt
...
 
-Your application needs to include a `wpe-build` script in the [`scripts`](https://docs.npmjs.com/cli/v8/using-npm/scripts) field in your project's `package.json` file. It should execute the build command for your project. This is the script that the Atlas platform will execute when it builds your app.
-
-For example, the `package.json` file for a Faust.js or Next.js app would need to be updated as shown below.
+1. adding a "wpe-build" script in the package.json file for your app.
```
Comment (kellenmace) [inline atlas/getting-started/deploy-from-existing-repo.mdx:17]: @dgallardox  I think we should capitalize the first letter here, remove the "ing" ending from "Adding", and add backticks to these inline code/file references, like this:
1. Add a `wpe-build` script in the `package.json` file for your app.

File: atlas/getting-started/deploy-from-existing-repo.mdx [hunk 3]
```diff
@@ -25,7 +23,12 @@ For example, the `package.json` file for a Faust.js or Next.js app would need to
   },
 ```
 
-If you don't know what command needs to be run to build your app, please refer to your JavaScript framework's documentation.
+3. creating a "ATLAS_BUILD_SCRIPT" environmental variable in the user-portal and adding the command as value. Learn more about environmental variables [here](#env-var).
```
Comment (kellenmace) [inline atlas/getting-started/deploy-from-existing-repo.mdx:26]: @dgallardox I think we should capitalize the first letter of this sentence and drop the "ing", put `ATLAS_BUILD_SCRIPT` in backticks since it's inline code, change "user-portal" to "User Portal" since it's a proper noun, and simplify the wording a bit. So this will become:

3. Create an `ATLAS_BUILD_SCRIPT` environmental variable in the User Portal, with the command to be run as its value. Learn more about environmental variables [here](#env-var).

File: atlas/getting-started/deploy-from-existing-repo.mdx [hunk 4]
```diff
@@ -25,7 +23,12 @@ For example, the `package.json` file for a Faust.js or Next.js app would need to
...
+
+![ATLAS_BUILD_SCRIPT env variable](./assets/img/env-variables.png)
+
+
+>If you don't know what command needs to be run to build your app, please refer to the JavaScript framework's documentation.
```
Comment (kellenmace) [inline atlas/getting-started/deploy-from-existing-repo.mdx:31]: @dgallardox I think we should say "your JavaScript framework's documentation" instead of "the JavaScript framework's documentation" here.

File: atlas/getting-started/deploy-from-existing-repo.mdx [hunk 5]
```diff
@@ -97,6 +100,7 @@ If you're using the Faust.js Starter Kit or an existing Faust.js app repository,
 
 ![First app deployed](./assets/img/first-app-deployed.png)
 
+<a name="env-var"></a>
```
Comment (kellenmace) [inline atlas/getting-started/deploy-from-existing-repo.mdx:103]: @dgallardox What's the purpose of this empty link tag? Was this left in on accident, maybe?

**Review by JEverhart383 — CHANGES_REQUESTED** [review]
@dgallardox everything looks good except that one last anchor tag. If you can delete that, I'll merge this into staging, and then if that looks good I'll ask you to open one additional PR into main from your branch. Thanks!

**Review by JEverhart383 — APPROVED** [review]
